src/morris-chart.directive.ts

```diff
@@ -46,6 +47,10 @@ export class MorrisChartDirective implements OnInit, AfterViewInit, OnChanges, O
     } else {
       this.ngZone.runOutsideAngular(() => {
         this.chartInstance = new this.window.Morris[this.type](this._options);
+        let my_this = this;
```

Comment:
You need to remove my_this and use arrow function please

        this.chartInstance.on('click', (i, row) => {
          this.clickChart.emit({ event, i, row });
        });